src/Event/IsPropagationStoppedCapableInterface.php

```diff
@@ -10,7 +10,7 @@
  * is marked as stopped after each listener is called.  If it is then it should
  * return immediately without calling any further Listeners.
  */
-interface StoppableEventInterface
+interface IsPropagationStoppedCapableInterface
```

Comment:
Why rename? I know that this name makes more sense, but I thought that the purpose of this fork was to support PHP 7.0. I don't see why the name needs to change. It just adds more distance between this fork and the original.

Reply:
I have no problem with distance, since there's no way changes from here will be merged to original PSR-14. The real reason, however, is that I needed to make an actual `StoppableEventInterface`, which extends an actual `EventInterface`. Didz me a conflict.

Reply:
Ah, understood.